src/Abp.Web.Api/WebApi/Controllers/Dynamic/Selectors/HttpVerbExtensions.cs

```diff
@@ -80,6 +82,11 @@ public static HttpVerb ToHttpVerb(this HttpMethod method)
                 return HttpVerb.Head;
             }
 
+            if (method == new HttpMethod("PATCH"))
```

Comment:
I've managed to find the class on [SymbolSource](https://www.symbolsource.org/Public/Metadata/NuGet/Project/HttpClient/0.5.0/Release/.NETFramework,Version%3Dv4.0/Microsoft.Net.Http/Microsoft.Net.Http/System/Net/Http/HttpMethod.cs?ImageName=Microsoft.Net.Http).
As you can see, both equality operator and Equals method are present.